extension/otto/otto.go

```diff
@@ -261,10 +263,11 @@ func (env *Environment) SetEventTimeLimit(eventRegex string, timeLimit time.Dura
 
 //Clone makes clone of the environment
 func (env *Environment) Clone() ext.Environment {
+	cloneMtx.Lock()
+	defer cloneMtx.Unlock()
 	clone := NewEnvironment(env.Name, env.DataStore, env.Identity, env.Sync)
-	interrupt := clone.VM.Otto.Interrupt
 	clone.VM.Otto = env.VM.Copy()
-	clone.VM.Otto.Interrupt = interrupt
+	clone.VM.Otto.Interrupt = make(chan func(), 1)
```

Comment:
here is a thread which describes the issue:
https://www.bountysource.com/issues/2289358-memleak-caused-by-otto-clones